src/helpers/label.ts

```diff
@@ -14,7 +17,7 @@ export const listLabelsForRepo = async (per_page?: number, page?: number): Promi
   });
 
   if (res.status === 200) {
-    return res.data.map((i) => i.name);
+    return res.data;
```

Comment:
This seems like a big change from the API where it apparently used to return a string, and then now its returning a `Label` object. Is this correct? Why is this only being changed now? I feel like the code in this revision and the last revision might not be compatible?

Reply:
They're compatible, just one file made use of this and now it's made compatible for easy use of the function